x-pack/filebeat/module/cyberarkpas/_meta/config.yml

```diff
@@ -0,0 +1,21 @@
+- module: cyberarkpas
+  audit:
+    enabled: true
+
+    # Set which input to use between udp (default), tcp, or file.
+    #
+    # var.input: udp
+
+    # var.syslog_host: localhost
+    # var.syslog_port: 9301
+
+    # With tcp input, set the optional tls configuration:
+    #var.ssl:
```

Comment:
Still didn't get TLS mode to work. With my current setup, Vault emits a TLS alert: bad certificate.